docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -954,15 +926,10 @@ desired={<Thirteen viewBox="140 40 680 500" />}
 
 <div>
 
-#### Current state
-
-Camunda 8 is reachable to the end-user but not yet exporting any data.
-
-Elasticsearch exporters are enabled for both regions, and it's ensured that the operation has finished.
-
-#### Desired state
-
-You are reactivating the existing exporters. This will allow Zeebe to export data to Elasticsearch again.
+| **Details**                 | **Current State**                                                       | **Desired State**                                  |
+| --------------------------- | ----------------------------------------------------------------------- | -------------------------------------------------- |
+| **Camunda 8 Accessibility** | Reachable to end-users, but currently not exporting any data.           | Reactivate existing exporters.                     |
```

Comment:
I think for the details you should align on same wording and just use `Camunda 8 Setup` or just go down to `Camunda 8`